### ISSUE TITLE
"grails create-demo" fails

### ISSUE THREAD

**Issue by roblight** [MayGo/grails-scaffold-angular#5]
Following the README.md under "Using plugin" to the letter, step 5 fails:

```
$ grails create-demo
| Packaging Grails application.....
Init bootstrap to generate build-test-data data
Bootstrap init done
| Generating application files from plugin /data/home/rwl/Sandbox/test/grails-angular/myApp/target/work/plugins/scaffold-angular-1.3.5 templates: 
Using templates dir: /data/home/rwl/Sandbox/test/grails-angular/myApp/target/work/plugins/scaffold-angular-1.3.5/src/templates/scaffold/
Using scaffold dirs from config:[backendRestGrailsApp:grails-app/, backendRestSrc:src/, backendRestTest:test/, frontendAngular:angular/]
Generating file: backendRestGrailsApp/dynamic/controllers/__packageName__/v1/__shortName__Controller.groovy
Generating file: backendRestGrailsApp/dynamic/services/__packageName__/__shortName__ModifyService.groovy
Generating file: backendRestGrailsApp/dynamic/services/__packageName__/__shortName__SearchService.groovy
Generating file: backendRestGrailsApp/dynamic/services/defpackage/TestDataGeneratorService.groovy
Generating file: backendRestGrailsApp/dynamic/conf/__UrlMappings.groovy
Generating file: backendRestGrailsApp/dynamic/conf/spring/__resources.groovy
Generating file: backendRestGrailsApp/dynamic/conf/TestDataConfig.groovy
Generating file: backendRestGrailsApp/dynamic/conf/__BuildConfig.groovy
Generating file: backendRestGrailsApp/dynamic/conf/__Bootstrap.groovy
| Error Error executing script CreateDemo: java.io.FileNotFoundException: /data/home/rwl/Sandbox/test/grails-angular/myApp/grails-app/conf/Bootstrap.groovy (No such file or directory) (Use --stacktrace to see the full trace)
```

**Comment by MayGo** [comment]
file myApp/grails-app/conf/Bootstrap.groovy exists?
On what operating system are you running it?

**Comment by roblight** [comment]
I'm on RHEL 6.7 and `Bootstrap.groovy` does not exist in the app.

**Comment by MayGo** [comment]
Bootstrap.groovy does not exist? There should be that file. Is it not
standard grails application?
23.12.2015 20:13 kirjutas kuupäeval "Rob Light" notifications@github.com:

**Comment by roblight** [comment]
I'm using Grails 2.4.3 with Java 8.  What versions are you using?

**Comment by roblight** [comment]
There isn't a `Bootstrap.groovy` in the app but there is a `BootStrap.groovy` file in `/conf` directory.

**Comment by roblight** [comment]
I downgraded to Java 7 with the same results.

**Comment by MayGo** [comment]
quick fix would be to rename 
myApp/target/work/plugins/scaffold-angular-1.3.5/src/templates/scaffold/backendRestGrailsApp/dynamic/conf/__Bootstrap.groovy
to __BootStrap.groovy.

I will fix it in next release, but it will take time. Because next release is going to use angular-material for scaffolding instead bootrap.

**Comment by roblight** [comment]
You must be on a Mac ;)  Glad to hear about the next release - looking forward to it.  Thanks!

**Comment by MayGo** [comment]
Released new version: ':scaffold-angular:1.4.2'
uses angular-material.
fixed this problem also.
